__tests__/components/editor/InputListLOC.test.js

```diff
@@ -95,4 +100,9 @@ describe('<InputList />', () => {
   it('sets the formData store with the total number or objects sent to "selected" when a change event happens', () => {
     expect(mockFormDataFn.mock.calls.length).toBe(2)
   })
+
+  it('gets the LOC lookup URL from the config based on the URI in useValuesFrom ', () => {
```

Comment:
This test would fail w/o the changes you made to the two source files below, I assume.

Reply:
Indeed:
```
 Expected: "https://id.loc.gov/vocabulary/frequencies"
    Received: undefined

      103 | 
      104 |   it('gets the LOC lookup URL from the config based on the URI in useValuesFrom ', () => {
    > 105 |     expect(wrapper.state('lookupUri')).toEqual(mockLookupConfig.uri)
          |                                        ^
```

Reply:
@jgreben 	Most excellent!

Reply:
If I change it to look at props instead of state I think this test should be moved to the parent component.

Reply:
nevermind. I figured out a better way. stay tuned.